game/room/room_repo.go

```diff
@@ -13,14 +13,14 @@ func NewRoomRepo(db *sql.DB) *RoomRepo {
 	return &RoomRepo{database: db}
 }
 
-func (rr *RoomRepo) RoomsByPlayerId(id int) []*Room {
+func (rr *RoomRepo) RoomsByPlayerId(playerID int) []*Room {
 	stmt, err := rr.database.Prepare("SELECT * FROM rooms WHERE owner_id = $1")
```

Comment:
Given you didn't opt for ORM's by default, I take it you're not a big fan of them? There is a tool to organize queries and generating code at the same named [sqlc](https://github.com/kyleconroy/sqlc). It allows you to have the nice sugary ORM/DAO-like APIs, without the [inherent performance drawbacks of ORMs](https://alanilling.com/exiting-the-vietnam-of-programming-our-journey-in-dropping-the-orm-in-golang-3ce7dff24a0f).

Reply:
This could definitely be something to look at. 

I find ORMs to not be as flexible as writing my own queries, even if that means taking on more work on my end. I think right now my plan is to get rooms loading in a somewhat hacked fashion and then once I have the foundation laid I want to refactor some to make everything consistent and clean.